File: database/port/Action.java
package database.port;

//[using]

@Table(name = "action")
class Action
{

    @Column(name = "valeur")
    public int valeur;    

    @Column(name = "id_action")
    public String idAction;    

    @Column(name = "nom")
    public String nom;    



    public void setValeur(int value){
        this.valeur = value;
    }
    public int getValeur(){
        return this.valeur;
    }

    public void setIdAction(String value){
        this.idAction = value;
    }
    public String getIdAction(){
        return this.idAction;
    }

    public void setNom(String value){
        this.nom = value;
    }
    public String getNom(){
        return this.nom;
    }

   
}

File: database/port/Action.cs
namespace database.port;

//[using]

[Table(name = "action")]
class Action
{

    [Column(name = "valeur")]
    public Int valeur { get; set; }    

    [Column(name = "id_action")]
    public String idAction { get; set; }    

    [Column(name = "nom")]
    public String nom { get; set; }    



    public void SetValeur(Int value){
        this.valeur = value;
    }
    public Int GetValeur(){
        return this.valeur;
    }

    public void SetIdAction(String value){
        this.idAction = value;
    }
    public String GetIdAction(){
        return this.idAction;
    }

    public void SetNom(String value){
        this.nom = value;
    }
    public String GetNom(){
        return this.nom;
    }

   
}
